src/ontology/cl-edit.owl

```diff
@@ -30947,15 +30948,15 @@ SubClassOf(obo:CL_4030039 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070006))
 
 # Class: obo:CL_4030040 (endometrial ciliated epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41591-020-1040-z") Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Glandular+epithelial+cells#endometrialciliatedcells") Annotation(oboInOwl:hasDbXref "PMID:31290979") Annotation(oboInOwl:hasDbXref "PMID:37999981") obo:IAO_0000115 obo:CL_4030040 "A multi-ciliated cell of the endometrial epithelium. This cell is characterized by the presence of motile cilia on its apical surface, which facilitates the movement of mucous across the endometrial surface.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://doi.org/10.1016/j.coemr.2024.100503>) Annotation(oboInOwl:hasDbXref "PMID:31290979") Annotation(oboInOwl:hasDbXref "PMID:37999981") Annotation(oboInOwl:hasDbXref "doi:10.1038/s41591-020-1040-z") Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Glandular+epithelial+cells#endometrialciliatedcells") obo:IAO_0000115 obo:CL_4030040 "A multi-ciliated cell of the endometrial epithelium. This cell is characterized by the presence of 9+2 motile cilia on its apical surface, which facilitates the movement of mucous across the endometrial surface.")
```

Comment:
Please, add doi reference in format doi:XXXXX instead of the URL.

Reply:
I think it would be good to explain what 9+2 means. Something like "... by the presence of 9+2 motile cilia, an arrengement consisting of nine fused pairings of microtubules forming a ring around two unfused central microtubules, on its apical surface, ...".

Reply:
The doi needs to be a literal, it is currently a IRI.

Reply:
Done